### PR TITLE
Support floor division assignment (//=)

### LINKED ISSUE #292: Missing floor division compound operator `//=`
I saw that the floor division operator was added recently, but the compound assign statement is missing the equivalent operator:

```lua
value //= 2
```

### PR DESCRIPTION
**Pull request by chriscerie**
Closes #292.

### CODE REVIEW

File: full-moon/src/ast/parsers.rs [hunk 1]
```diff
@@ -2260,6 +2260,7 @@ make_op_parser!(CompoundOp, ParseCompoundOp,
         MinusEqual,
         StarEqual,
         SlashEqual,
+        DoubleSlashEqual,
```
Comment (Kampfkarren) [inline full-moon/src/ast/parsers.rs:2263]: Standard is `TwoSlashEqual`

**Review by Kampfkarren — CHANGES_REQUESTED** [review]

File: full-moon/src/ast/parsers.rs (hunk 1 above)
Comment (chriscerie) [reply full-moon/src/ast/parsers.rs:2263]: It’d be ideal if it’s consistent with its counterpart `DoubleSlash`. Should that be changed to `TwoSlash` then?

**Review by jeparlefrancais — APPROVED** [review]
Thank you for jumping on this 🥇

File: full-moon/src/ast/parsers.rs (hunk 1 above)
Comment (JohnnyMorganz) [reply full-moon/src/ast/parsers.rs:2263]: Ah i made a mistake when naming double slash originally. That's annoying...

Yeah I think it's better to stay related to DoubleSlash. Renaming to TwoSlash is a breaking change

Comment (Kampfkarren) [reply full-moon/src/ast/parsers.rs:2263]: Fair enough then don't change anything but we should deprecate both at some point

**Pull request merged**